docs/conf/portland/2023/welcome-wagon.rst

```diff
@@ -3,5 +3,195 @@
+
+Welcome Wagon Tours
+~~~~~~~~~~~~~~~~~~~
+
+Sunday at noon
+Sunday at 2:00pm
+Monday morning at 8:15am
+
+Join us for an informal Introduction to Write the Docs in the Library/Astoria Room. We’ll review information about the conference specifically for first-timers and give everyone a chance to meet someone new. Come on a short tour of the venue so you’ll know where everything is and everything you can take part in.
+
+Pro Tips
+~~~~~~~~
+
+-  You don't need to go to every talk. Look through the schedule of events before you arrive or while you are eating or taking a break. Figure out which talks you want to see the most. Spread out your time between talks, unconference sessions, networking, and breaks.
+-  Speaking of breaks--conferences are exhilarating, but can also be exhausting. Give your brain a break! Grab a quiet spot in the Sunset Room or take a quick walk. Play a board game on your lunch break. Come back invigorated.
+-  Starting Monday morning, check the unconference schedule in the Library/Astoria Room to see if there are any sessions you are interested in attending. New sessions are added all the time, so check back periodically.
+-  Eat! You can use the energy. There will be snacks in the Library/Astoria Room.
+-  Are you looking for a job or is there an opening at your company? Check out the job board and the job fair in the Library/Astoria Room.
+
+.. contents::
+   :local:
+   :backlinks: none
+
+Where is everything?
+~~~~~~~~~~~~~~~~~~~~
+
+-  The conference main stage and lightning talks are in Revolution Hall on the second floor.
+-  The unconference takes place on the first floor in the Library/Astoria Room.
+-  The Job Fair is on the first floor in the Library/Astoria Room.
+-  Snacks and drinks are on the first floor in the Library/Astoria Room.
+-  The quiet room is on the second floor in the Sunset Room.
+
+
+How should I get around?
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Portland is a very walk-able town.
+-  There are good public transportation options and taxi services. Check out the `Visiting Portland section of the Write the Docs website <https://www.writethedocs.org/conf/portland/2023/visiting/>`__ for more info.
+
+How should I dress?
+~~~~~~~~~~~~~~~~~~~
+
+-  Portland is a casual-dress town and so is the Write the Docs conference. You'll be meeting business colleagues at this conference, though, so neat and comfortable are good dress guidelines.
+-  If you are going on the `Write the Docs hike on Saturday <https://www.writethedocs.org/conf/portland/2023/outing/>`__, be sure
+   to bring appropriate hiking clothes and shoes. This time of year, the Pacific Northwest tends to be muddy or raining with occasional swaths of blue skies. Layering is usually the way to go.
+
+What will I eat?
+~~~~~~~~~~~~~~~~
+
+-  Snacks are provided for you on conference days.
+-  Coffee, tea, and water are always available in the Crystal Ballroom. Bring a water bottle to make it easier for you to stay hydrated.
```

Comment:
s/Crystal Ballroom/(how are we describing the Show Bar refreshment area?)

Reply:
Thank you! According to the notes, it sounds like snacks will be in Library Room. Do you mean the fact that you could buy your own refreshments from Show Bar? I'm not sure when that is open. I thought it was only open during shows.